libs/sdk-model/src/notificationChannels/index.ts

```diff
@@ -0,0 +1,51 @@
+// (C) 2022-2024 GoodData Corporation
+
+/**
+ * @alpha
+ */
+export interface ICreateWebhookDefinition extends Omit<IWebhookDefinition, "id"> {
```

Comment:
I think we don't have `Create` prefix anywhere in the codebase yet. Could we stick to current conventions? Eg for measure metadata object it's: `IMeasureMetadataObjectBase` / `IMeasureMetadataObject` / `IMeasureMetadataObjectDefinition`